rest_framework/compat.py

```diff
@@ -82,7 +82,7 @@ def apply_markdown(text):
         )
         md_filter_add_syntax_highlight(md)
         return md.convert(text)
-except ImportError:
+except (ImportError, AttributeError):
```

Comment:
what's the traceback you get that raises AttributeError? markdown has support for 3.12 and doesn't call `entry_points(...).get(...)`

Reply:
it was fixed in https://github.com/Python-Markdown/markdown/commit/7919bfd8cf44e4fb50a3ccbd82b725db34b66a08#diff-73515afd0109793374fb1748b8642ce5099d18d6924a8a06420da6e219eea32cL88

Reply:
you'll need to update the pin in https://github.com/encode/django-rest-framework/blob/0abb84fa3955956fc6156cc2c51222cd4ceae783/requirements/requirements-optionals.txt#L7

Reply:
check related issue for traceback, markdown is already set to 3.3

Reply:
being handled here https://github.com/encode/django-rest-framework/pull/9181